cadc-registry/src/main/java/ca/nrc/cadc/reg/Capabilities.java

```diff
@@ -89,55 +87,41 @@
  *
  * @author yeunga
  */
-public class Capabilities
-{
+public class Capabilities {
+
     private static Logger log = Logger.getLogger(Capabilities.class);
 
     private final List<Capability> capabilities = new ArrayList<Capability>();
 
     /**
      * Constructor.
      */
-    public Capabilities()
-    {
+    public Capabilities() {
     }
 
     /**
      * Find all associated capabilities.
      *
      * @return all associated capabilities.
      */
-	public List<Capability> getCapabilities()
-	{
-		return this.capabilities;
-	}
-
-	/**
-	 * Find the capability associated with the specified standard identifier.
-	 *
-	 * @param standardID standard identifier for the required capability
-	 * @return capability found or null if not found
-	 */
-	public Capability findCapability(final URI standardID)
-	{
-		boolean found = false;
-		Capability retCap = null;
-
-		for (Capability cap : this.capabilities)
-		{
-			if (cap.getStandardID().equals(standardID))
-			{
-				if (found)
-				{
-					String msg = "Matched more than one capability";
-					throw new RuntimeException(msg);
-				}
+    public List<Capability> getCapabilities() {
+        return capabilities;
+    }
 
-				found = true;
-				retCap = cap;
-			}
-		}
+    /**
+     * Find the capability associated with the specified standard identifier. This method
+     * returns the first matching capability.
+     *
+     * @param standardID standard identifier for the required capability
+     * @return the first matching capability or null if not found
+     */
+    public Capability findCapability(final URI standardID) {
+        for (Capability cap : this.capabilities) {
+            if (cap.getStandardID().equals(standardID)) {
+                return cap;
+            }
+        }
 
-		return retCap;
-	}
+        return null;
+    }
 }
```

Comment:
Yes, I think this is okay. Having a capability repeated is surely a config error.